### ISSUE TITLE
Plot changes even when run is already finished

### ISSUE THREAD

**Issue by prash-p** [wandb/client#520]
* Weights and Biases version: 0.8.9
* Python version: 3.6.5
* Operating System: Linux

### Description

Metrics that I log and then plot (training loss, validation loss, etc. against step or epoch) change values every time the plot is 'updated', even when a run is already finished (so no new values are being logged). So when I am plotting metrics which are from completed runs, the plot keeps changing every few seconds. Why does this happen?

**Comment by vanpelt** [comment]
If you log more than 10k steps we randomly sample from them for performance reasons.  This random sampling will cause the changing you're seeing.  We're working on an overhaul of our backend to atleast do the sampling deterministically.  In the meantime you can always get the entire set of points using our python API, and if you reduce the frequency you're logging metrics this will be less evident.

**Comment by ariG23498** [comment]
Hey @prash-p 
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.
